app/views/admin/loans/index.html.slim

```diff
@@ -0,0 +1,3 @@
+- content_for(:title, content='Loans')
```

Comment:
This was to replace `- content_for :title do t("general.coops") end`. Ruby recommended syntax `- content_for :title { t("general.coops") }` does not work here. Do you have a prefered way to display this text. My preference is `do` `end`.